docs/subvolume.md

```diff
@@ -23,8 +24,9 @@ kubectl rook-ceph subvolume ls
 # Filesystem  Subvolume  SubvolumeGroup  State
 # ocs-storagecluster-cephfilesystem csi-vol-427774b4-340b-11ed-8d66-0242ac110004 csi in-use
 # ocs-storagecluster-cephfilesystem csi-vol-427774b4-340b-11ed-8d66-0242ac110005 csi in-use
-# ocs-storagecluster-cephfilesystem csi-vol-427774b4-340b-11ed-8d66-0242ac110006 csi in-use
+# ocs-storagecluster-cephfilesystem csi-vol-427774b4-340b-11ed-8d66-0242ac110006 csi snapshot-retained
```

Comment:
You can skip listing subvolume which is already deleted but retained only due to snapshot, WDYT?

Reply:
Yeah seems a better option to be. I will update it.